README.md

```diff
@@ -78,14 +78,17 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-## Deploy on CodeSandbox
+## Deploy on Deno Deploy
 
-1. Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin)
-2. Return to the `express-guest-list-api-memory-data-store` GitHub repository and add `box` between `github` and `.com` in the URL (`githubbox.com` instead of `github.com`) - this creates a [synced template](https://codesandbox.io/docs/learn/devboxes/synced-templates) on CodeSandbox
-3. Click on "Fork" at the top right of the page - this creates a devbox in your account from the synced template
-4. Beside "Fork", click "Share", select "Public" from the "Visibility" dropdown and click "Move Devbox" - this allows everybody read access to your devbox
+1. Click "Fork" at the top right of the `express-guest-list-api-memory-data-store` GitHub repository page, select your GitHub account and click "Fork" to create a copy of the GitHub repository in your account
+2. On [Create a New Project](https://dash.deno.com/new_project), click on "Continue with GitHub", authorize Deno Deploy to access your GitHub account and accept the terms and conditions
+3. In the "Select User or Organization" dropdown, select "Add GitHub Account", which will open an authorization page on GitHub
```

Comment:
```suggestion
3. In the "Select User or Organization" dropdown, select "Add GitHub Account" if you do not already see your account - this will open an authorization page on GitHub
```